ex4B/ex4Spec.js

```diff
@@ -0,0 +1,142 @@
+var toNumber = require('./ex4.js');
+
+describe("toNumber", function() {
+
+	///////////////////////////////////////////////////////////////////////	
+	// Numbers
+	it("converts single digit", function() {
+		expect(toNumber("5")).toBe(5);
+	});
+	
+	it("converts 2 digits", function() {
+		expect(toNumber("71")).toMatch(71);
+	});
+
+	it("converts many digits", function() {
+		expect(toNumber("712387328182")).toMatch(712387328182);
+	});
+	
+	///////////////////////////////////////////////////////////////////////	
+	// Numbers negative
+	it("converts a single negative digit", function() {
```

Comment:
single digit is 3, negative number is -17581. You probably meant `a negative number with a single digit`. Shortening the sentence makes it incorrect.